pkg/reporter/reporter_test.go

```diff
@@ -221,3 +221,46 @@ func TestJSONReporterPublishNoViolations(t *testing.T) {
 		t.Errorf("expected %q, got %q", `{"violations":[]}`, buf.String())
 	}
 }
+
+//nolint:paralleltest
+func TestGitHubReporterPublish(t *testing.T) {
+	// Can't use t.Parallel() here because os.Setenv() forbids that
```

Comment:
🤔 os.Setenv forbids it, but does t.Setenv, too?

Reply:
Oh, yes that it does — the comment was wrong. I'll fix, thanks!